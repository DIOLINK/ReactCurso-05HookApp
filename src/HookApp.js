import React from 'react';

const HookApp = () => {
  return (
    <div>
      <h1>Hola Roger</h1>
    </div>
  );
};

export default HookApp;
