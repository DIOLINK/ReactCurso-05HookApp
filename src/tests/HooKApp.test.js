import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('Probando HookApp.js', () => {
  it('Debe mostrar el componente', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
