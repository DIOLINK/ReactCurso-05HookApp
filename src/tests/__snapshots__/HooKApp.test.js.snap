// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando HookApp.js Debe mostrar el componente 1`] = `
<div>
  <h1>
    Hola Roger
  </h1>
</div>
`;
